compiler/one-cmds/tests/onecc_024.test

```diff
@@ -70,7 +70,7 @@ LUCI_LOG=5 onecc -C ${configfile} -OONECC_024 > ${filename}.log 2>&1
 
 clean_envir
 
-if grep -q "MakeBatchNormGammaPositivePass" "${filename}.log"; then
+if ! grep -q "MakeBatchNormGammaPositivePass" "${filename}.log"; then
```

Comment:
Q) why added `!` ?

Reply:
The priority of options is `group option < individual option in cfg option`.

```bash
$ cat OONECC_024.cfg

[one-optimize]
make_batchnorm_gamma_positive=True

$ cat onecc_024.cfg

[one-optimize]
input_path=inception_v3.onecc_024.circle
output_path=inception_v3.opt.circle
make_batchnorm_gamma_positive=False 
```

So, `make_batchnorm_gamma_positive` option should be turned off.